w1r3/java/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.19 AS build
+
+# Install the minimal development tools:
+RUN apk update && \
+    apk add maven openjdk11
```

Comment:
I recommend using `maven:3.9.6-eclipse-temurin-11` as the build images instead of raw alpine and installing dependencies.

Reply:
Done